src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const ulMain = document.querySelector('ul');
+const listUl = ulMain.children;
```

Comment:
```suggestion
const listUl = [...ulMain.children];
```
To avoid double work on 7 and 17 lines